unattended_scripts/install_functions/opendistro/wazuh-passwords-tool.sh

```diff
@@ -181,12 +252,12 @@ createBackUp() {
 generatePassword() {
 
     if [ -n "${NUSER}" ]; then
-        logger_pass "Generating random password"
-        PASSWORD=$(< /dev/urandom tr -dc _A-Z-a-z-0-9 | head -c${1:-32};echo;)
+        logger "Generating random password"
```

Comment:
Change `logger` to `logger_pass` so it can be used individually

Reply:
Done 21e827b